datasource/remote/src/main/java/com/egdroid/datasource/remotemodule/MovieResponse.java

```diff
@@ -0,0 +1,63 @@
+package com.egdroid.datasource.remotemodule;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public abstract class MovieResponse {
+
+    @SerializedName("results")
+    public List<Results> results;
+    @SerializedName("total_pages")
+    public int totalPages;
+    @SerializedName("total_results")
+    public int totalResults;
+    @SerializedName("page")
+    public int page;
+
+    public static class Results {
```

Comment:
Please add some documentation or a more descriptive name to make it clear what this class does